### DISCUSSION TITLE
Voice issue after joining the room 1/2 minutes

### DISCUSSION THREAD

**Discussion by elvis0920** [mozilla/hubs#5754] [Hubs Cloud Questions] [answered]
Hi,
I'm hosting HC on AWS and it's running perfectly for the first 1/2 minutes. However, shortly after, the voice begins to shake and drag, it's like a slow-motion, then it's not able to continue to talk anymore. 
Anyone facing the same issue here, or anyone know what the problem might be and how to fix it?

**Comment by elvis0920** [comment, marked as answer]
This issue/bug has been spotted before at issue #2854, but I could find any follow-ups. Did I miss anything?

**Comment by elvis0920** [comment]
This issue has been reported for quite a few times from my users. I've also checked with AWS and their team run some tests and replied that their cloud/service are fine. They suggested to check with Hubs. @matthewbcool Would you be able to spare some time on this issue please? Thank you.

**Reply by matthewbcool** [reply]
more specifics in a bug report (webrtc logs, console logs, network conditions ect), reproducible on hubs.mozilla.com would be helpful. 

Looks like the main issue is documented here: https://github.com/mozilla/hubs/issues/2445
Thanks for flagging.
